### DISCUSSION TITLE
Selenium deactivate Headless mode  &   Send Key List

### DISCUSSION THREAD

**Discussion by Burningx2** [openbullet/openbullet#849] [General]
Hey 
Is there any way deactivate Crome Selenium Headless mode 
in OpenNullet 2 0.2.3 It Should be possible Cant Ceck the Fails.

And is there any KEY list? 
Dont know how Send Key TAB

**Comment by openbullet** [comment]
OB1 is not supported anymore, this is the OB1 repo

**Reply by Burningx2** [reply]
Thge question be the same using openbullet 2

**Comment by openbullet** [comment]
Then please open it on the official OB2 forum and not on the OB1 github repo 😅 I will answer
https://discourse.openbullet.dev/
